src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
public enum Color {
```